interface/reports/sales_by_item.php

```diff
@@ -221,7 +223,7 @@ function thisLineItem($patient_id, $encounter_id, $rowcat, $description, $transd
         <?php } ?>
   <td class="detail">
             <?php if ($GLOBALS['sales_report_invoice'] == 0 || $GLOBALS['sales_report_invoice'] == 2) { ?>
-   <a href='../patient_file/pos_checkout.php?ptid=<?php echo attr_url($patient_id); ?>&enc=<?php echo attr_url($encounter_id); ?>'>
+   <a href='../patient_file/pos_checkout.php?ptid=<?php echo attr_url($patient_id); ?>&enc=<?php echo attr_url($encounter_id); ?>' target='_blank'>
```

Comment:
Need to add following attribute `rel='noopener'` for security reasons (basically locks out javascript from trying to do anything to the frame that this new frame originates from)
```
<a href='../patient_file/pos_checkout.php?ptid=<?php echo attr_url($patient_id); ?>&enc=<?php echo attr_url($encounter_id); ?>' target='_blank' rel='noopener'>

Reply:
did find the below at https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener 
![Screenshot from 2025-02-05 12-32-35](https://github.com/user-attachments/assets/9d8c4349-78f6-45ba-b200-9639c851d904)

Reply:
nice! Sound like not needed since an <a> element.